requirements.txt

```diff
@@ -0,0 +1,8 @@
+click>=7,<8
+requests>=2,<3
+pymongo>=3.10,<4
+dnspython>=1.16,<2
+pyyaml>=5,<6
+tabulate>=0.8,<0.9
+junitparser>=1,<2
+numpy
```

Comment:
I still don't understand fully when the `stats` CLI command is to be used but if we are to use it on Evergreen, we will also need to add `numpy` to the `install_requires` list in `setup.py`.

Reply:
Done.